source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ContainerRenderer.java

```diff
@@ -170,20 +182,64 @@ public static void ApplyBleed(CollectionTypeElement collectionElement, ViewGroup
         }
     }
 
-    public static void ApplyPadding(ContainerStyle elementContainerStyle, ContainerStyle parentContainerStyle, ViewGroup collectionElementView, Context context, HostConfig hostConfig)
+    /**
+     * @deprecated Separated into specific {@link #applyPadding} and {@link #applyContainerStyle}.
+     */
+    public static void ApplyPadding(ContainerStyle computedContainerStyle, ContainerStyle parentContainerStyle, ViewGroup collectionElementView, HostConfig hostConfig)
+    {
+        applyPadding(computedContainerStyle, parentContainerStyle, collectionElementView, hostConfig);
+        applyContainerStyle(computedContainerStyle, collectionElementView, hostConfig);
+    }
+
+    public static void applyPadding(ContainerStyle computedContainerStyle, ContainerStyle parentContainerStyle, ViewGroup collectionElementView, HostConfig hostConfig)
+    {
+        applyPadding(computedContainerStyle, parentContainerStyle, collectionElementView, hostConfig, false);
+    }
+
+    public static void applyPadding(ContainerStyle computedContainerStyle, ContainerStyle parentContainerStyle, ViewGroup collectionElementView, HostConfig hostConfig, boolean hasBorder)
```

Comment:
we may have to figure out at some point if we want static methods to start with a lower case or an upper case letter

Reply:
Java/Kotlin convention is to start with lowercase

Reply:
I think I got confused because of the deprecated method above